source/release-notes/2.9.0.rst

```diff
@@ -11,11 +11,16 @@
 is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL
 performance.
 
+Highlights
+==========
+
+This release brings a major rework of the Query Analytics (QAN) component, completing the migration from Angular to React, and adding new UI functionality and features. You can read more in the accompanying blog post (`here < https://www.percona.com/blog/2020/07/13/qan-release-post/>`_).
```

Comment:
@PaulJacobs-percona , could you also add here the links to QAN epics ( https://jira.percona.com/browse/PMM-5516, https://jira.percona.com/browse/PMM-5125 ) as we discussed.

Reply:
Ok will test they work with :jirabug: and add them.